src/main/scala/Countdown.scala

```diff
@@ -0,0 +1,63 @@
+object Countdown {
+
+  def vowel(): String = {
+    val r = scala.util.Random
+    val letterID = r.nextInt(4)
+    letterID match {
+      case 0 => "A"
+      case 1 => "E"
+      case 2 => "I"
+      case 3 => "O"
+      case 4 => "U"
+    }
+  }
+
+  def consonant():String= {
+    val r = scala.util.Random
```

Comment:
r is not a great name
this line is also duplicate to line 4
I think there's an easier way rather than having those 20 case statements

Reply:
('A' to 'Z').toList(Random.nextInt(20)) something along these lines